### PR TITLE
Enriched right sidebar

### PR DESCRIPTION
**Pull request by zdmytriv**
## what
* Added github buttons on right side

## why
* Improvement usability

## references
*

### CODE REVIEW

File: src/theme/sections/github-toc-buttons/github-toc-buttons.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+              data-size="large"
+              aria-label="Open GitHub">GitHub</GitHubButton>
+          </li>
+
+          <li>
+            <GitHubButton
+              href={'https://github.com/' + repoFullName + '/issues'}
+              data-size="large"
+              data-show-count="true"
+              aria-label="Issue cloudposse/terraform-aws-components on GitHub">Issues</GitHubButton>
```
Comment (osterman) [inline src/theme/sections/github-toc-buttons/github-toc-buttons.js:28]: Should "cloudposse/terraform-aws-components" be hardcoded?

File: src/theme/sections/github-toc-buttons/github-toc-buttons.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          </li>
+
+          <li>
+            <GitHubButton
+              href={'https://github.com/' + repoFullName + '/fork'}
+              data-icon="octicon-repo-forked"
+              data-size="large"
+              data-show-count="true"
+              aria-label="Fork cloudposse/terraform-aws-components on GitHub">Forks</GitHubButton>
```
Comment (osterman) [inline src/theme/sections/github-toc-buttons/github-toc-buttons.js:37]: Should "cloudposse/terraform-aws-components" be hardcoded?

File: src/theme/sections/github-toc-buttons/github-toc-buttons.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          </li>
+
+          <li>
+            <GitHubButton
+              href={'https://github.com/' + repoFullName}
+              data-icon="octicon-star"
+              data-size="large"
+              data-show-count="true"
+              aria-label="Star buttons/github-buttons on GitHub">Stars</GitHubButton>
```
Comment (osterman) [inline src/theme/sections/github-toc-buttons/github-toc-buttons.js:46]: Should "cloudposse/terraform-aws-components" be hardcoded?

**Comment by zdmytriv** [comment]
Closing in favor of https://github.com/cloudposse/docs/pull/558

**Pull request closed without merging**